core/bourbon/helpers/_gamma.scss

```diff
@@ -0,0 +1,21 @@
+@charset "UTF-8";
+
+/// Perform gamma correction on a single color channel.
```

Comment:
Yes, I also thought about that. Unfortunately, I am still very confused about the exact terminology (see for example [this discussion](https://github.com/less/less.js/issues/2829)). So instead of getting the details wrong I just skipped them.

If you have a good idea just go ahead.